bin/projects/dbatools/dbatools/Parameter/DbaInstanceParameter.cs

```diff
@@ -244,12 +244,12 @@ public DbaInstanceParameter(string Name)
             string tempString = Name;
 
             // Handle and clear protocols. Otherwise it'd make port detection unneccessarily messy
-            if (Regex.IsMatch(tempString, "^TCP:", RegexOptions.IgnoreCase))
+            if (Regex.IsMatch(tempString, "^TCP:", RegexOptions.IgnoreCase)) //TODO: Use case insinsitive String.BeginsWith()
```

Comment:
Why?

Reply:
I'll write some code to prove it, but I'm pretty sure Regexes are a lot slower than simpple substring calls.

Reply:
REgardless, I'll touch that later.